hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/util/EnricherInternalExceptionType.java

```diff
@@ -0,0 +1,15 @@
+package org.hypertrace.traceenricher.util;
+
+public enum EnricherInternalExceptionType {
+  PROCESS_EXCEPTION("processing.exception");
```

Comment:
We still have single one. Do we plan to add more enum types?

Reply:
As discussed offline, for now we will not have a separate exception tag. We will have a single counter for all internal exceptions within an enricher and later if needed we will handle this